Balance/Table Cells/CryptoBalanceTableViewCell.swift

```diff
@@ -287,7 +288,21 @@ private class CryptoRow: UIView {
             make.top.equalTo(iconImageView).offset(-2)
         }
 
-        tokenNameLabel.text = name
+        var tokenName = "No Name"
+
+        if let name = name {
```

Comment:
Just FYI, if we wanted to let iOS handle it, we can set a property on the UILabel to add an ellipsis. This solution works too if we want to hardcode a certain number of chars.

Reply:
https://developer.apple.com/documentation/uikit/nslinebreakmode/bytruncatingtail

Reply:
Oh sweet. I'll merge this for now as a few people have complained about the 0 tokens. I am not super familiar with changing parts of a pull request.

Reply:
@ricburton in the future, to change stuff in your open PRs, just make new commits to the branch and push them